libraries/chain/include/eosio/chain/block_state_legacy.hpp

```diff
@@ -78,7 +78,7 @@ namespace eosio::chain {
          _cached_trxs = std::move( trxs_metas );
       }
 
-      bool                                                validated = false;
+      copyable_atomic<bool>                               validated{false};
```

Comment:
I don't think we should make this change on `release/1.0`

Reply:
Why not?

Reply:
It is private and will serialize the same way. So, I guess I'm ok with changing this.